client/my-sites/subscribers/subscriber-list/subscriber-list.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslate } from 'i18n-calypso';
+import FormCheckbox from 'calypso/components/forms/form-checkbox';
+import { SubscriberRow } from './subscriber-row';
+import { Subscriber } from '../types';
+import './styles.scss';
+
+type SubscriberListProps = {
+	subscribers: Subscriber[];
+	locale: string;
+};
+
+export const SubscriberList = ( { subscribers, locale }: SubscriberListProps ) => {
+	const translate = useTranslate();
+	return (
+		<ul className="subscriber-list" role="table">
+			<li className="row header" role="row">
+				<span className="subscriber-list__checkbox-column" role="columnheader">
+					<FormCheckbox />
+				</span>
+				<span className="subscriber-list__profile-column" role="columnheader">
+					{ translate( 'Name' ) }
+				</span>
+				<span className="subscriber-list__subscription-type-column" role="columnheader">
+					{ translate( 'Subscription type' ) }
+				</span>
+				<span className="subscriber-list__rate-column" role="columnheader">
+					{ translate( 'Open Rate' ) }
+				</span>
+				<span className="subscriber-list__since-column" role="columnheader">
+					{ translate( 'Since' ) }
+				</span>
+				<span className="subscriber-list__menu-column" role="columnheader"></span>
```

Comment:
In this case, I think using two different selectors for rows and list is making it a bit confusing on Sass (sorry for the bad suggestion 😬), maybe using just the `subscriber-list__` as a namespace would be cleaner and also reusable on both components (`SubscriberList` and `SubscriberRow`). For the specificities of each column you could use additional selectors for now, like `checkbox-column`. Another thing is that you can include a `hidden` selector just for the hidden columns, it adds some clarity on what is rendered without even look to the css.

```
		<ul className="subscriber-list" role="table">
			<li className="subscriber-list__row header" role="row">
				<span className="subscriber-list__column checkbox-column hidden" role="columnheader">
					<FormCheckbox />
				</span>
				<span className="subscriber-list__column profile-column" role="columnheader">
					{ translate( 'Name' ) }
				</span>
				<span className="subscriber-list__column hidden" role="columnheader">
					{ translate( 'Subscription type' ) }
				</span>
				...
```